src/api/json/catalog.json

```diff
@@ -4660,6 +4660,12 @@
         "2.15.0": "https://json.schemastore.org/pantsbuild-2.15.0.json"
       }
     },
+    {
+      "name": "Unity Package",
+      "description": "Unity 3D package file",
+      "fileMatch": ["package.json"],
```

Comment:
This is a file name that is already used by "npm". Please find a more unique name for the `fileMatch` property.